build-tests/api-extractor-scenarios/etc/exportImportStarAs2/api-extractor-scenarios.api.md

```diff
@@ -9,10 +9,6 @@
 // @public (undocumented)
 function exportedApi(): forgottenNs.ForgottenClass;
 
-// @public (undocumented)
-class ForgottenClass {
-}
-
```

Comment:
I think we want this change. `ForgottenClass` is exported by `forgottenNs` **but** `forgottenNs` isn't actually exported itself. Thus, `ForgottenClass` isn't exported and thus shouldn't appear in the API report. I believe previously `ForgottenClass` was marked as consumable when it in fact was not (this was a bug).